main.py

```diff
@@ -59,4 +59,14 @@ async def InsertRetroactiveLeave(ctx, discorduser, leavetype, startdate, enddate
     await ctx.author.send(content = message_content)
     await ctx.message.delete()
 
+@slash.slash(name = "IsMemberOnLeave", description = "Checks if the required member is on leave for the requested day", options = UI.CreateIsMemberOnLeaveOptions(), guild_ids = guild_ids)
+async def IsMemberOnLeave(ctx, discorduser, date):
+    is_on_leave, reason = leave_interface.IsMemberOnLeave(discorduser.id, date)
+    if Utilities.IsAdmin(ctx.author):
+        await ctx.author.send(content = str(is_on_leave) + ", " + reason)
+    else:
+        await ctx.author.send(content = is_on_leave)
```

Comment:
Still conflicted with sending a dm for this. I feel like sending it in the channel itself is much cleaner. But dealing with showing the reason is annoying. 

Few options here, 
1. only dm when an admin requests
2. only dm when a parameter is passed in (that means no reason even if admin requests if it's not a dm)
3. send the result to the channel but dm the reason to the admin

can you quickly consider ups and downs for these?

Reply:
Last option is of course dm always

Reply:
the whole idea of sending a DM is to keep the commands channel as clean as possible we don`t want the channel to get over flooded with request if ppl trying to know who is going to be on leave, especially with the upcoming feature of using range you will get separated messages for each member on leave, which will send more than 1 message of pretty big content in the channel and this will be messy. Another reason for sending the DM is to hide any sensitive information  the users enters and that's why we delete the command completely after usage